pyproject.toml

```diff
@@ -37,6 +43,21 @@ mypy = "^0.782"
 pytest-cov = "^2.10.1"
 pytest-structlog = "^0.3"
 coverage = {extras = ["toml"], version = "^5.3"}
+Sphinx = "^4.0.2"
+m2r2 = "^0.2.7"
+sphinx-rtd-theme = "^0.5.2"
+
+[tool.poetry.extras]
+docs = [
+    "sphinx",
+    "m2r2",
+    "toml",
+    "sphinx-rtd-theme",
+    "pydantic",
+    "structlog",
+    "colorama",
+    "dataclasses"
```

Comment:
I wouldn't think these ones would need to be listed under extras if they're a non-optional core dependency?

Reply:
Discussed these are needed for poetry/pip integration into RTD.